bazel/README.md

```diff
@@ -149,20 +149,13 @@ set different options. See below to configure test IP versions.
 
 ## Linking against libc++ on Linux
 
-To link Envoy against libc++, use the following commands:
+To link Envoy against libc++, follow the [quick start](#quick-start-bazel-build-for-developers) to setup Clang+LLVM and run:
 ```
-export CC=clang
-export CXX=clang++
 bazel build --config=libc++ //source/exe:envoy-static
```

Comment:
I think what I was wondering before is whether it's possible to just have `--config-libc++` be the default?

The issue that I have found in the past is that I don't think there is a way to have a default config and then actual have a local config disable it and replace it with something else?

/wait-any

Reply:
I will recommend having `--config=libc++` in `user.bazelrc` file to make it default. It is possible to override but not easy, partially because toolchains are also defaulted to libstdc++.

Reply:
OK, should we document that somewhere?